stellargraph/mapper/graphwave_generator.py

```diff
@@ -17,34 +17,36 @@
 from tensorflow.keras import backend as K
 import numpy as np
 from ..core import StellarGraph
+from ..core.validation import require_integer_in_range
 from scipy.sparse.linalg import eigs
 from scipy.sparse import diags
-from ..core.experimental import experimental
 
 
-@experimental(
-    reason="lacks unit tests, and the time complexity could be reduced using Chebyshev polynomials.",
-    issues=[815, 853],
-)
 class GraphWaveGenerator:
     """
-    Implementation of the GraphWave structural embedding algorithm from the paper:
-    "Learning Structural Node Embeddings via Diffusion Wavelets" (https://arxiv.org/pdf/1710.10321.pdf)
+    Implementation of the GraphWave structural embedding algorithm from the paper: "Learning Structural Node Embeddings
+    via Diffusion Wavelets" (https://arxiv.org/pdf/1710.10321.pdf)
 
     This class is minimally with a StellarGraph object. Calling the flow function will return a tensorflow
     DataSet that contains the GraphWave embeddings.
 
     This implementation differs from the paper by removing the automatic method of calculating scales. This method was
-    found to not work well in practice, and replicating the results of the paper requires manually specifying much larger
-    scales than those automatically calculated.
+    found to not work well in practice, and replicating the results of the paper requires manually specifying much
+    larger scales than those automatically calculated.
 
     Args:
         G (StellarGraph): the StellarGraph object.
         scales (iterable of floats): the wavelet scales to use. Smaller values embed smaller scale structural
             features, and larger values embed larger structural features.
+        degree: the degree of the Chebyshev polynomial to use. Higher degrees yield more accurate results but at a
+            higher computational cost. According to [1], the default value of `20` is accurate enough for most
```

Comment:
Minor but `'s are special in reStructuredText (which is what sphinx uses, not markdown). Isolated backticks like the 20 here are "interpreted text", and seem to become italicised with our current settings. If you're actually looking for italics, I think this would be clearer as:
```suggestion
            higher computational cost. According to [1], the default value of *20* is accurate enough for most
```

If you're looking for a inline code formatting, it's double-`:

```suggestion
            higher computational cost. According to [1], the default value of ``20`` is accurate enough for most
```

However, I suspect it's ok for this number to be just a number:

```suggestion
            higher computational cost. According to [1], the default value of 20 is accurate enough for most
```

(I was referring to https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst)